www/assets/css/theme.css

```diff
@@ -1,28 +1,56 @@
+/* ╭─────────────────────────────────────────────────────────╮
+ * │                         Palette                         │
+ * ╰─────────────────────────────────────────────────────────╯
+ */
+:root {
+  --bg-main: #fff6f6;     /* <body> background color */
+  --bg-dark: #ccc;     /* <html> background color */
+  --bg-alt1: #efefef;  /* mostly used by header */
+  --bg-alt2: #dfdfdf;  /* mostly used by <kbd> */
+  --fg-main: #333;     /* text color */
+  --fg-alt:  #544;     /* used by "ergol rocket", banner and <code> */
+  --accent:  #d63;     /* mostly used by titles and <a> */
+  --bg-hl:   #d634;    /* accent color, but with less opacity */
```

Comment:
I think the taxonomy could be a lot better.

- `--bg-dark` is not a good name for `#ccc` at all, maybe `--bg-outer` or `--bg-root` ?
- `hl` and `h1` are too close, `highlight` would be more readable
- not sure about alt/alt1/alt2 either, maybe `--{fg,bg}-banner` and `--bg-key` or something ?
- `--accent` makes little sense, `--fg-em` or `--fg-h2` or `--fg-link` mayby ?

Unrelated : `--bg-main` and `--fg-main` should be grouped together to help keeping the contrast in mind (important a11y principle).

Reply:
Thanks for the suggestion. I tried to use common color names I found while browsing neovim themes, but most of those were just `fg-alt` or `bg4`.

Reply:
This might be a stupid suggestion, but what about using `--bg-em` instead of `--bg-hl` ? Since we are highlighting something with that color to put *emphisis* on it ?

Reply:
that could work, yes

Reply:
On second thought if we use `--bg-em` here, we should *not* use `--fg-em` for the link color. And '--fg-em` was probably a very bad idea anyway, `--fg-link` would be straight-forward (and we could change it later for `--fg-h2` if we add a color for h3).

Reply:
To put it another way : `--bg-banner` + `--fg-banner` would work because these two bg/fg colors are used on the same element and grouping them helps keeping the contrast in mind — even if one of these two colors is used elsewhere as well. But `--bg-em` + `--fg-em` wouldn’t make sense because `-em` never refers to the same element here.